webextensions/api/browsingData.json

```diff
@@ -302,9 +302,7 @@
                 "firefox": {
                   "version_added": "84"
                 },
-                "firefox_android": {
-                  "version_added": false
-                },
+                "firefox_android": "mirror",
```

Comment:
I'm wondering if it would make sense to leave a note to mention that contextualIdentities feature is not available and so using this option on Android makes more sense if used for the default and private cookieStoreId values.

Reply:
done